geomesa-raster/src/test/scala/org/locationtech/geomesa/raster/data/RasterBoundsTableTest.scala

```diff
@@ -0,0 +1,129 @@
+/*
+ * Copyright 2014 Commonwealth Computer Research, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.locationtech.geomesa.raster.data
+
+import org.junit.runner.RunWith
+import org.locationtech.geomesa.raster.util.RasterUtils
+import org.locationtech.geomesa.utils.geohash.BoundingBox
+import org.specs2.mutable.Specification
+import org.specs2.runner.JUnitRunner
+
+@RunWith(classOf[JUnitRunner])
+class RasterBoundsTableTest extends Specification{
+  sequential
+
+  var testIteration = 0
+
+  val wholeWorld = BoundingBox(-180.0, 180.0, -90.0, 90.0)
+
+  def getNewIteration() = {
+    testIteration += 1
+    s"testRBTT_Table_$testIteration"
+  }
+
+  "RasterStore" should {
+    "return the bounds of a empty table as the whole world" in {
+      val tableName = getNewIteration()
+      val theStore = RasterUtils.createRasterStore(tableName)
+
+      // get bounds
+      val theBounds = theStore.getBounds()
+
+      theBounds must beAnInstanceOf[BoundingBox]
+      theBounds must beEqualTo(wholeWorld)
```

Comment:
Hmm... I think I may have suggested that empty tables return the whole world. 

I wish there were another obvious option.  If someone can think of a sentinel empty bounds value, shout.